src/test/java/core/basesyntax/FileServiceTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.CouldNotReadFromFileException;
+import core.basesyntax.service.FileService;
+import core.basesyntax.service.impl.FileServiceImpl;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class FileServiceTest {
+    private static final String INPUT_FILE = "src/test/test_resources/inputTest.csv";
+    private static final String INVALID_FILE = "broken";
+    private static final String OUTPUT_FILE = "src/test/test_resources/outputTest.csv";
+    private static final String TEST_DATA = "Some new sentence";
+    private static FileService fileService;
+
+    @BeforeAll
+    static void initAll() {
```

Comment:
It is better to create objects under test in beforeEach to make sure that tests fully independent (check other tests)